assets/helpers/internationalisation/__tests__/countryGroupTest.js

```diff
@@ -2,31 +2,173 @@
+    });
+    expect(detect()).toEqual('EURCountries');
 
+    jsdom.reconfigure({
+      url: 'https://support.theguardian.com/int',
     });
+    expect(detect()).toEqual('International');
+  });
 
-    it('returns \'null\' for a country that does not exist', () => {
+  it('should return the correct country group from the query parameter', () => {
 
-      expect(fromCountry('42')).toBeNull();
+    jsdom.reconfigure({
+      url: 'https://support.theguardian.com/examplePath?countryGroup=GBPCountries',
+    });
+    expect(detect()).toEqual('GBPCountries');
 
+    jsdom.reconfigure({
+      url: 'https://support.theguardian.com/examplePath?countryGroup=UnitedStates',
     });
+    expect(detect()).toEqual('UnitedStates');
+
+    jsdom.reconfigure({
+      url: 'https://support.theguardian.com/examplePath?countryGroup=AUDCountries',
+    });
+    expect(detect()).toEqual('AUDCountries');
+
+    jsdom.reconfigure({
+      url: 'https://support.theguardian.com/examplePath?countryGroup=EURCountries',
+    });
+    expect(detect()).toEqual('EURCountries');
+
+    jsdom.reconfigure({
+      url: 'https://support.theguardian.com/examplePath?countryGroup=International',
+    });
+    expect(detect()).toEqual('International');
+  });
+
+  it('should return the correct country group from GU_country cookie', () => {
+
+    jsdom.reconfigure({
+      url: 'https://support.theguardian.com/examplePath',
+    });
+
+    document.cookie = 'GU_country=UK';
+    expect(detect()).toEqual('GBPCountries');
+
+    document.cookie = 'GU_country=US';
+    expect(detect()).toEqual('UnitedStates');
+
+    document.cookie = 'GU_country=AU';
+    expect(detect()).toEqual('AUDCountries');
+
+    document.cookie = 'GU_country=FR';
+    expect(detect()).toEqual('EURCountries');
+
+    document.cookie = 'GU_country=CI';
+    expect(detect()).toEqual('International');
+
+
+    jsdom.reconfigure({
+      url: 'https://support.theguardian.com/examplePath?countryGroup=42',
+    });
+    document.cookie = 'GU_country=BR';
+    expect(detect()).toEqual('International');
+  });
+
+  it('should return the correct country group from GU_country cookie', () => {
+
+    jsdom.reconfigure({
+      url: 'https://support.theguardian.com/examplePath',
+    });
+
+    document.cookie = 'GU_country=UK';
+    expect(detect()).toEqual('GBPCountries');
+
+    document.cookie = 'GU_country=GB';
+    expect(detect()).toEqual('GBPCountries');
+
+    document.cookie = 'GU_country=US';
+    expect(detect()).toEqual('UnitedStates');
+
+    document.cookie = 'GU_country=AU';
+    expect(detect()).toEqual('AUDCountries');
+
+    document.cookie = 'GU_country=FR';
+    expect(detect()).toEqual('EURCountries');
+
+    document.cookie = 'GU_country=CI';
+    expect(detect()).toEqual('International');
+
+
+    jsdom.reconfigure({
+      url: 'https://support.theguardian.com/examplePath?countryGroup=42',
+    });
+    document.cookie = 'GU_country=BR';
+    expect(detect()).toEqual('International');
+  });
```

Comment:
Isn't this a duplicate of the test above, but with GB added in too?

Reply:
Yes I will remove it

Reply:
Removed in: ce0940c5d793fd9daf979c4092348b77bd6fdc16